async_firebase/messages.py

```diff
@@ -395,3 +410,62 @@ def success_count(self):
     @property
     def failure_count(self):
         return len(self.responses) - self.success_count
+
+
+class ErrorInfo:
```

Comment:
Do you know if this class can only be used with topic management responses? If so, I recommend changing the name to something more specific, like `TopicManagementErrorInfo`.

Reply:
It`s better to rename. Done.